modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-precache-feature_{context}"]
+= Using the container image pre-cache feature
+
+The SNO spoke clusters might have limited bandwidth to the container image registry, which can cause a timeout before the updates are completed. 
+The pre-cache feature allows the required container images to be present on the spoke cluster before the update maintenance time.
+
+[NOTE]
+====
+The maintenance window is not set by the Topology Aware Lifecycle Operator. You can apply the Topology Aware Lifecycle Operator CR at the beginning of the maintenance window by manual application or by external automation. The update procedure start when the `enable` field is set to `true`.
+====
+
+[id="talo-precache-create_policy_{context}"]
+== Creating a policy to pre-cache container images
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Extract the container image in digest format for the `PolicyGenTemplate` CR by running the following command:
++
+[source,terminal]
+----
+$ oc adm release info 4.9.18 | head
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:      4.9.18
+Digest:    sha256:1234abcd5678efgh
+Created:   2022-01-26T14:50:25Z
+OS/Arch:   linux/amd64
+Manifests: 522
+
+Pull From: example.io/openshift-release/ocp-release@sha256:1234abcd5678efgh
+----
+
+. Create the `PolicyGenTemplate` CR in `inform` mode, then save the YAML in the `du-upgrade.yaml` file.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
```

Comment:
Same as above, I think we should use the actual Policy object here for reference and link to a "using PolicyGenTemplate to create policies" section.